scripts/missions/asa/03_That_Which_Curdles_Blood.lua

```diff
@@ -0,0 +1,112 @@
+-----------------------------------
+-- A Shantotto Ascension
+-- A Shantotto Ascension M3
+-----------------------------------
+-- !addmission 11 2
+-- Kuroido-Moido : !pos -113.78 -3.25 105.40 240
+-- Faulpie       : !pos -178.88 -1 9.89 230
+-- Abd-al-Raziq  : !pos 126.7 2.0 -0.23 234
+-- Trodden Snow  : !pos -19.7 -17.3 104.4 126
+-----------------------------------
+require('scripts/globals/missions')
+require('scripts/settings/main')
```

Comment:
Appears to be unused, also, since there are keyitems references, should include `require('scripts/globals/keyitems')`